monorepo/monorepo-ws/apps/federation-gateway/src/main.ts

```diff
@@ -1,14 +1,15 @@
-import { LocalStorageManager } from './services/local-storage-manager';
-import { SupergraphStorageManager } from './services/interfaces';
+
 import { ServerManager } from './services/server-manager';
+import { Storage, SupergraphManager } from '@monorepo-ws/supergraph-manager';
+import { createSupergraphManager } from './dependencies';
 
-const main = async () => {
-  const superGraphManager: SupergraphStorageManager = new LocalStorageManager()
-  const serverManager: ServerManager = new ServerManager(superGraphManager)
+const main = async (storage: Storage) => {  
+  const supergraphManager: SupergraphManager = createSupergraphManager(storage)
+  const serverManager: ServerManager = new ServerManager(supergraphManager)
   await serverManager.runApp()
 }
 
-main()
+main(Storage.Local)
```

Comment:
💡 tady uz jde krasne udelat to ze to jestli se vytvori ta appka s lokal nebo prismou na zaklade env promenne 

`main(config.storageType ? Storage.Local : Storage.Prisma)`

Reply:
Uděláno úplně jinak, toto si řeší libka. Ještě pred tím si rovnou apka zkontroluje hodnotu z konfigurace a převede na enum.